mara_description/urdf/mara.urdf.xacro

```diff
@@ -280,39 +280,28 @@
       <selfCollide>True</selfCollide>
     </gazebo>
 
+    <!-- dummy link with the required tags-->
     <link name="${prefix}motor6_link">
       <inertial>
-        <origin rpy="0 0 0" xyz="0.0 0.0 -0.0173"/>
-        <mass value="0.14"/>
-        <inertia ixx="9.22e-05" ixy="6.35e-7" ixz="2.00e-11" iyy="9.73e-5" iyz="2.36e-9" izz="1.77e-4"/>
+        <origin rpy="0 0 0" xyz="0 0 0"/>
```

Comment:
why you removed this inertias?

Reply:
This is just a dummy link.
This link was the gripper adapter. Which does not make sense to me and if we want to launch just MARA without gripper it should not have the gripper adapter i guess.